### DISCUSSION TITLE
How do I/can I use Outliner plug-in?

### DISCUSSION THREAD

**Discussion by EricEWeir** [vslinko/obsidian-outliner#424] [Q&A]
I have the plug-in. I don’t have a clue how to start using it. How do I? 

I had a long happy productive experience with a powerful command-based outliner back in the days of MS-DOS. I have found nothing that can match its power since.

I’m willing to settle for less, as long as I can use it to organize and reorganize actual content, to draft long form pieces. Limited to creating abstract sketchy outlines it would not be helpful.

Looks to me doubtful that it can be used in mobile apps, since they don’t—Apple’s don’t—have the keys needed for hot keys. Is that correct?

**Comment by vslinko** [comment]
Hello @EricEWeir 

To use the plugin, you need to create a deeply structured list and use [predefined comands](https://github.com/vslinko/obsidian-outliner#features).

Almost all specific commands have a Mobile Quick Action alternative that can be added above your mobile device's keyboard. I also added command icons to the README for easier navigation.

**Reply by EricEWeir** [reply]
Thank you very much. Pretty straightforward. A little trying things it would’ve become clear.
